pkg/mermaidsequencediagram/mermaid.go

```diff
@@ -0,0 +1,79 @@
+package mermaidsequencediagram
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/anz-bank/sysl/pkg/sysl"
+)
+
+const projDir = "../../"
+
+func GenerateMermaidSequenceDiagram(m *sysl.Module, appname string, epname string,
+	previousapp string, indent int, thestart bool) (string, error) {
```

Comment:
`thestart` is a bool but every test case you have sets it to `true` - do you think it's worth getting rid of this parameter to simplify the API?

Reply:
thestart is set to true only once from the beginning of the function. It is very much needed cos I am recursively calling the same function and I would want to know when I've reach the start again for implementing the call/return to "the world".

Reply:
Check line 32